src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxKinds.cs

```diff
@@ -4,6 +4,8 @@
 
 #nullable enable
 
+using Microsoft.CodeAnalysis.LanguageServices;
+
 namespace Microsoft.CodeAnalysis.CSharp
```

Comment:
same concern.  Can this be under a different namespace?  basically, i don't want it to be the cae that anything in nested namespaces sees these guys by default.

Reply:
Sure, this type was already in this namespace prior to my change, but it seems reasonable to fix it in this PR.

Reply:
Done with the next commit. Note that I have not moved `CSharpSyntaxFactsService` and `VisualBasicSyntaxFactsService` into a specialized namespace as I am going to make both these as private nested types of the corresponding language service factories in a follow-up PR.